src/System.Management.Automation/namespaces/RegistryWrapper.cs

```diff
@@ -17,29 +17,31 @@
 
 namespace Microsoft.PowerShell.Commands
 {
+
+#nullable enable
     internal interface IRegistryWrapper
     {
-        void SetValue(string name, object value);
+        void SetValue(string? name, object value);
 
-        void SetValue(string name, object value, RegistryValueKind valueKind);
+        void SetValue(string? name, object value, RegistryValueKind valueKind);
 
         string[] GetValueNames();
 
         void DeleteValue(string name);
 
         string[] GetSubKeyNames();
 
-        IRegistryWrapper CreateSubKey(string subkey);
+        IRegistryWrapper? CreateSubKey(string subkey);
```

Comment:
Is null returned when create fails?

Reply:
From docs: https://docs.microsoft.com/en-us/dotnet/api/microsoft.win32.registrykey.createsubkey?view=net-5.0#Microsoft_Win32_RegistryKey_CreateSubKey_System_String_
> Returns
RegistryKey
The newly created subkey, or null if the operation failed. If a zero-length string is specified for subkey, the current RegistryKey object is returned.